packages/nav-frontend-alertstriper/md/Alertstripe.overview.mdx

```diff
@@ -30,8 +30,8 @@ import Alertstripe from "./..";
 >
 <Alertstripe type="feil"
   >Innsending av søknad er nede, vennligst prøv igjen senere. Se
```

Comment:
Burde ikke `>` her være på linje 32? Samme for linjer 66/65.

Reply:
🤦 Fikset..